roles/build_containers/tasks/main.yml

```diff
@@ -90,6 +90,13 @@
     - cifmw_build_containers_buildah_push | default ('false') | bool
     - not cifmw_build_containers_push_containers | bool
 
+- name: Return containers built, tag, and artifacts location
```

Comment:
Can we also add a task here to call ta.yaml task file with different conditional like cifmw_build_containers_retag so that they can retag it with something different tag?

Reason: periodic-container-tcib-build-push-centos-10-master tags the container with hash, I want to reuse this playbook to retag it with current in the same.